worker/rpc_task.go

```diff
@@ -1,22 +1,56 @@
 package worker
 
 import (
+	"context"
+	"github.com/ohsu-comp-bio/funnel/config"
 	pbf "github.com/ohsu-comp-bio/funnel/proto/funnel"
 	"github.com/ohsu-comp-bio/funnel/proto/tes"
+	"google.golang.org/grpc"
 	"time"
 )
 
 // TODO document behavior of slow consumer of task log updates
 
 // RPCTask provides access to writing task logs over gRPC to the funnel server.
 type RPCTask struct {
-	client *schedClient
-	taskID string
+	client        *taskClient
+	taskID        string
+	updateTimeout time.Duration
+}
+
+func newRPCTask(conf config.Worker, taskID string) (*RPCTask, error) {
+	client, err := newTaskClient(conf)
+	if err != nil {
+		return nil, err
+	}
+	return &RPCTask{client, taskID, conf.UpdateTimeout}, nil
+}
+
+func (r *RPCTask) Task() (*tes.Task, error) {
+	return r.client.GetTask(context.Background(), &tes.GetTaskRequest{
+		Id:   r.taskID,
+		View: tes.TaskView_BASIC,
+	})
+}
+
```

Comment:
The worker probably doesn't need the full view, since that would just give it extra logs and add network overhead. The default is the minimal view, which is all `State()` needs.